src/Sylius/Component/Locale/Context/LocaleContext.php

```diff
@@ -32,6 +33,59 @@ class LocaleContext implements LocaleContextInterface
      */
     protected $storage;
 
+    /**
+     * Meta data for special languages,
+     * by default we assume that all languages are LTR except mentioned otherwise.
+     *
+     * TODO This list should be completed
+     *
+     * @var array
+     */
+    protected static $localeMetaData = array(
```

Comment:
Unfortunately I couldn't find anything on the web in a quick attempt, I'll look into ICU project to see if I can find something if not maybe we should extract this out?!